src/components/playground/ColorInput.js

```diff
@@ -61,34 +65,47 @@ function ColorInput(props) {
         flex: '1',
       }}
     >
-      <TextField
-        variant="standard"
+      <FormControl
         error={!isValid}
-        multiline={isGradient}
-        label={label}
-        value={value}
-        onChange={handleChange}
         sx={{
           textTransform: 'capitalize',
           width: 1,
         }}
-        helperText={!isValid && 'Invalid syntax'}
-        InputProps={{
-          endAdornment: (
+        variant="standard"
+      >
+        <InputLabel
+          sx={{
+            width: '130%',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            <span>{label}</span>
+            <Tooltip title="Reset to Default">
+              <RestartAlt onClick={handleReset} color="error" />
```

Comment:
Could you change the `cursor` to a `pointer` to give a more clear indication of a button